rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/service_type_support.h

```diff
@@ -7,6 +7,24 @@ typedef struct service_type_support_callbacks_t
 {
   const char * package_name;
   const char * service_name;
+  // Function to create a requester
+  void* (*create_requester)(void * participant, const char * service_name,
+                            void **reader);
```

Comment:
Should the newly created requester also be passed as a `**`? It seems odd to me to return one object as the return value and another as a `**` parameter. Usually when there is multiple return values, we either need to encapsulate in a structure or both are returned through `**` parameters and a return code is returned. Any thoughts on this?